posthog/models/person/person.py

```diff
@@ -99,3 +100,39 @@ class Meta:
 
     # current version of the id, used to sync with ClickHouse and collapse rows correctly for new clickhouse table
     version: models.BigIntegerField = models.BigIntegerField(null=True, blank=True)
+
+
+class PersonOverrideManager(models.Manager):
+    """A manager for PersonOverride that sets the is_long_term flag on creation."""
+
+    def create_override(self, *args: Any, long_term_cutoff: int = 45, **kwargs: Any) -> "PersonOverride":
```

Comment:
I'm not aware of how things will look (or already look) from the ingestion side. But if we already have some logic there to calculate `is_long_term` or if we plan to, then yeah I can drop this and just expose the flag for ingestion to set.

Reply:
The alternative would be to remove the ingestion side of things and keep this. Again, I don't know how things look from that end so I can't make the call of which one to get rid of.

Maybe @tiina303 also has some input on this?

Reply:
Seems like we'll just be using `old_person_created_at` (now renamed to `oldest_event`) and set that during ingestion. So the model need only expose the parameter. I've updated the PR :+1:

Reply:
Just to explain how it would have worked on ingestion side: we'd need to send the kafka events to one of the other table in CH, which would mean different kafka topics, so the ingestion code would need to know that. That said the code for figuring it if short or long term could have still lived here and we'd just get the row after we updated and based on what value we pick the kafka topic to use (this column/value could have even been the name of the kafka topic to make it simpler).